src/Core/src/Platform/Windows/SwitchExtensions.cs

```diff
@@ -19,20 +21,18 @@ public static void UpdateTrackColor(this ToggleSwitch toggleSwitch, ISwitch view
 				return;
 			}
 
-			if (view.TrackColor == null)
-			{
-				return;
-			}
+			var trackColor = view.TrackColor?.ToPlatform() ?? new SolidColorBrush(Color.FromArgb(6, 0, 0, 0));
```

Comment:
Why use `Color.FromArgb(6, 0, 0, 0)`? Ideally, we could use a resource, the color used by default by the platform control (like ToggleSwitchContainerBackground).

Reply:
I have set the default value to match the ToggleSwitchFillOff value because it is [updated](https://github.com/dotnet/maui/pull/25068/files#diff-b27c75bf6f5be6960b5cd922f31a2a4d4632c3c61b3099f23efe8bec7156855bR32) whenever ToggleSwitchFillOff changes.

![screenshot](https://github.com/user-attachments/assets/1033ba65-7054-45d8-b6ed-6212889480a0)